### DISCUSSION TITLE
Does this plugin track query parameters?

### DISCUSSION THREAD

**Discussion by notflip** [johannschopplich/nuxt-gtag#5] [Q&A]
I had issues with the classic vue-gtag plugin, it only sends the name of the route to Google Analytics.
I'm wondering if this plugin will track more context such as the query string.

**Comment by johannschopplich** [comment]
Excellent question. As of right now, I'm not sure how Google handles history state changes. Technically speaking, queries should be part of the history object and thus appear in the Google Analytics log. 

![grafik](https://user-images.githubusercontent.com/27850750/226282217-f3fce03e-3a69-47f5-91ab-0fee5463dba3.png)

We should just test, how Google handles them and what the History object looks like:

![grafik](https://user-images.githubusercontent.com/27850750/226377636-fc39e91d-0838-4447-8bc1-2e5b36a26281.png)

****

[📚 Image sources](https://www.analyticsmania.com/post/single-page-web-app-with-google-tag-manager/)
